views/TopStories/index.jsx

```diff
@@ -50,6 +80,15 @@ export default React.createClass({
     this.setState({ showQuery: false });
   },
 
+  onChangeSort(e) {
+    this.setState({ sortType: e.target.value });
+    let sortVal = e.target.value == 'date' ? '-blekko.chrondate,-_score' : '';
```

Comment:
now that I think about it - would it be better to pass 'date' or 'relevance' to the `query.sort` value? then in the `query-builder.js` you do this same test. will keep the notion of fields in the `query-builder.js` instead of scattered in the `TopStories` component